src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import ArticleCard from "./ArticleCard";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://project-northcoders-nc-news.herokuapp.com/api/articles`)
+      .then((res) => res.json())
+      .then((data) => {
+        const allArticles = data;
+        setArticles(allArticles);
+      });
+  }, []);
+
+  return (
+    <main>
+      <h2>Articles</h2>
+      <ul>
+        {articles.map((article) => {
+          return <ArticleCard key={article.article_id} articles={article} />;
```

Comment:
Do we not need to see who wrote it? Maybe the time it was wrote? The comment counts? At the moment it just looks like blocks of text which is not very informative of what time text is.